cfgov/regulations3k/migrations/0021_rm_hero_links.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-03-05 18:45
+from __future__ import unicode_literals
+
+from django.db import migrations
+import wagtail.wagtailcore.blocks
+import wagtail.wagtailcore.fields
+import wagtail.wagtailimages.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('regulations3k', '0021_rm_hero_links'),
```

Comment:
Something happened here with your update. This needs to be pointing to regulations3k migration 0020.

Reply:
Fixed